docs/api/api.yml

```diff
@@ -55,6 +54,8 @@ paths:
           content:
             'application/pdf': {}
             'application/html': {}
+        400:
+          description: The input JSON could not be parsed correctly.
         500:
           $ref: "#/components/responses/InternalServerError"
```

Comment:
I'm not familiar enough with the syntax here, but what is the difference between the `$ref` here and the newly added `description` above. Shouldn't there be a description here as well? If so, make it consistent.